### ISSUE TITLE
segmentation fault (core dumped)

### ISSUE THREAD

**Issue by GuHuangAI** [MohameHassan/prox#22]
I do fitting on the quantitative dataset, but when i run:
`python ./prox/main.py --config cfg_files/RGB.yaml --recording_dir /export/home/lg/data/prox/quantitative/recordings/vicon_03301_01 --output_folder ./RGB_results --vposer_ckpt /export/home/lg/data/prox/quantitative/models/vposer_v1_0/ --part_segm_fn /export/home/lg/data/prox/quantitative/models/smplx_parts_segm.pkl --model_folder /export/home/lg/data/prox/quantitative/models`, 
The output code:
`Processing: /export/home/lg/data/prox/quantitative/recordings/vicon_03301_01/Color/s001_frame_00001__00.00.00.023.jpg
Found Trained Model: /export/home/lg/data/prox/quantitative/models/vposer_v1_0/snapshots/TR00_E096.pt
/export/home/lg/.local/opt/anaconda3/envs/torch10/lib/python3.6/site-packages/smplx/body_models.py:280: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  param[:] = torch.tensor(params_dict[param_name])
Camera initialization done after 16.8501
Camera initialization final loss 721.0235
Stage 000 done after 33.7366 seconds
Stage 001 done after 5.8872 seconds
Stage 002 done after 65.9812 seconds
Orientation:   0%|                                                                                                                                                                    | 0/1 [01:45<?, ?it/s[1]    6203 segmentation fault (core dumped)  python ./prox/main.py --config cfg_files/RGB.yaml --recording_dir                                                                | 3/5 [01:45<01:19, 39.56s/it]`
What happened and how to solve this problem?
My environment:
CentOS7
GCC 7.3.0
python 3.6.13
torch 1.1.0 ( the version of torch in the requirements is torch==1.0.1.post2, but the human_body_prior package needs torch>=1.1)
